OpenSim/Common/ComponentConnector.h

```diff
@@ -133,9 +134,18 @@ class OSIMCOMMON_API AbstractConnector : public Object {
     /** Disconnect this Connector from the connectee object */
     virtual void disconnect() = 0;
 
+protected:
+    const Component& getOwner() const { return _owner.getRef(); }
+    void setOwner(const Component& o) { _owner.reset(&o); }
```

Comment:
I'm adding an analogous method in Outputs to fix the output copying bug #809. However, I wrote it to take a `const Component*` instead of a reference. I think it's more natural that way, and results in less `&`'s and `*`'s. Whatever we do we should be consistent. So let me know what you think we should go with.